cmd/serverless-init/tag/tag.go

```diff
@@ -11,12 +11,56 @@ import (
 	"strings"
 )
 
-func GetBaseTags() []string {
-	if len(os.Getenv("K_SERVICE")) > 0 && len(os.Getenv("K_REVISION")) > 0 {
-		return []string{
-			fmt.Sprintf("revision:%s", strings.ToLower(os.Getenv("K_REVISION"))),
-			fmt.Sprintf("service:%s", strings.ToLower(os.Getenv("K_SERVICE"))),
+type tagPair struct {
+	name    string
+	envName string
+}
+
+func getTag(envName string) (string, bool) {
+	value := os.Getenv(envName)
+	if len(value) == 0 {
+		return "", false
+	}
+	return strings.ToLower(value), true
+}
+
+func GetBaseTagsMap() map[string]string {
+	tags := map[string]string{}
+	listTags := []tagPair{
+		{
+			name:    "cloudrunrevision",
+			envName: "K_REVISION",
+		},
+		{
+			name:    "cloudrunservice",
+			envName: "K_SERVICE",
```

Comment:
What's the value of K_SERVICE usually? Does it make sense as a default for service if DD_SERVICE isn't specified?

Reply:
K_SERVICE is the name of the cloud run service
It does perfect sense to use this value as DD_SERVICE if not set
I'll address that in another PR :100: